ideas/58-mainnet.md

```diff
@@ -0,0 +1,71 @@
+## Preamble
+
+    Idea: 58-mainnet-default
+    Title: Use Mainnet as the default network
+    Status: In progress
+    Created: 2017-12-08
+
+
+## Summary
+Alpha builds of Status should default to Mainnet. We should be using Status in a real-world environment.
+
+## Swarm Participants
+
+- Lead Contributor: @adambabik
+- Testing & Evaluation: <!-- @username -->
+- Contributor: @themue
+- Contributor: <!-- @username -->
+- PM: @chadyj 
+- UX (if relevant): <!-- @username -->
+
+## Product Overview
+
+Currently the Status alpha builds use testnet, but using Mainnet is a key milestone for the public launch of Status. There will be potentially be major performance and security implications so these issues should be surfaced as soon as possible so the app can be tested in a real-world environment. 
+
+From a users perspective, Mainnet should be used as it is required by most popular DApps, and is needed for real transactions, so the full potential of Status can't be seen until we switch.
+
```

Comment:
Even if the title "Add Mainnet" says we're adding it I would like to make more clear here that the other networks are still able to select. And it also would be interesting if it is intended that Mainnet will become default or has to be selected explicitly (together with a possible warning) while we're in test.